app/models/vm_or_template/right_sizing.rb

```diff
@@ -38,12 +38,12 @@ module VmOrTemplate::RightSizing
   module ClassMethods
     DEFAULT_CPU_RECOMMENDATION_MINIMUM = 1
     def cpu_recommendation_minimum
-      VMDB::Config.new("vmdb").config.fetch_path(:recommendations, :cpu_minimum) || DEFAULT_CPU_RECOMMENDATION_MINIMUM
+      Settings.recommendations.try(:cpu_minimum) || DEFAULT_CPU_RECOMMENDATION_MINIMUM
```

Comment:
Thanks.  Keenan gave me the heads up about this in another PR.  I just grep'd around the project for similar things because I wasn't sure if I should have added them myself, but I prefer that over a `.try()`.  Will do.